store/sgxLevelDB/sgxLeveldb.go

```diff
@@ -0,0 +1,251 @@
+// Package sgxLevelDB implements a light client level db for panacea-doracle.
+// It does include Set & Get functions that are sealed & unsealed in the sgx environment.
+
+package sgxLevelDB
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/medibloc/panacea-doracle/sgx"
+	log "github.com/sirupsen/logrus"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/errors"
+	"github.com/syndtr/goleveldb/leveldb/opt"
+	"github.com/syndtr/goleveldb/leveldb/util"
+	tmdb "github.com/tendermint/tm-db"
+)
+
+var (
+	// errBatchClosed is returned when a closed or written batch is used.
+	errBatchClosed = errors.New("batch has been written or closed")
+
+	// errKeyEmpty is returned when attempting to use an empty or nil key.
+	errKeyEmpty = errors.New("key cannot be empty")
+
+	// errValueNil is returned when attempting to set a nil value.
+	errValueNil = errors.New("value cannot be nil")
+)
+
+type dbCreator func(name string, dir string) (tmdb.DB, error)
+
+var backends = map[tmdb.BackendType]dbCreator{}
+
+func registerDBCreator(backend tmdb.BackendType, creator dbCreator, force bool) {
+	_, ok := backends[backend]
+	if !force && ok {
+		return
+	}
+	backends[backend] = creator
+}
+
+func init() {
+	dbCreator := func(name string, dir string) (tmdb.DB, error) {
+		return NewGoLevelDB(name, dir)
+	}
+	registerDBCreator(tmdb.GoLevelDBBackend, dbCreator, false)
+}
+
+type GoLevelDB struct {
+	Db *leveldb.DB
+}
+
+var _ tmdb.DB = (*GoLevelDB)(nil)
+
+func NewGoLevelDB(name string, dir string) (*GoLevelDB, error) {
+	return NewGoLevelDBWithOpts(name, dir, nil)
+}
+
+func NewGoLevelDBWithOpts(name string, dir string, o *opt.Options) (*GoLevelDB, error) {
+	dbPath := filepath.Join(dir, name+".db")
+	db, err := leveldb.OpenFile(dbPath, o)
+	if err != nil {
+		return nil, err
+	}
+	database := &GoLevelDB{
+		Db: db,
+	}
+	return database, nil
+}
+
+// Get implements DB.
+func (db *GoLevelDB) Get(key []byte) ([]byte, error) {
+	if len(key) == 0 {
+		return nil, errKeyEmpty
+	}
+	res, err := db.Db.Get(key, nil)
+	if err != nil {
+		if err == errors.ErrNotFound {
+			return nil, nil
+		}
+		return nil, err
+	}
+	// unseal data
+	unsealedRes, err := sgx.Unseal(res, true)
+	log.Debugf("Unseal value from key : %X\n", key)
+	if err != nil {
+		return nil, err
+	}
+
+	return unsealedRes, nil
+}
+
+// Has implements DB.
+func (db *GoLevelDB) Has(key []byte) (bool, error) {
+	bytes, err := db.Get(key)
+	if err != nil {
+		return false, err
+	}
+	return bytes != nil, nil
```

Comment:
think it can be easily modified like this.
```suggestion
	return db.DB.Has(key)
```